cli-plugins/insight/cli-overview.hbs.md

```diff
@@ -6,9 +6,16 @@ Follow the below steps to install, configure, and use the Tanzu Insight CLI plug
 
 >**Note:** Prior to using the CLI plug-in, you must install the Supply Chain Security Tools - Store, either as its own package, or as part of Tanzu Application Platform Build profile or Tanzu Application Platform View profile.
 
-1. [Install the Tanzu Insight CLI plug-in](cli-installation.md)
-1. [Configure target endpoint and certificate](../../scst-store/using-encryption-and-connection.md)
-1. [Configure access tokens](../../scst-store/create-service-account-access-token.md)
+1. If the insight plug-in is not already installed, see [Install the Tanzu Insight CLI plug-in](cli-installation.md)
+1. Configure target endpoint and certificate
+
+    Note: The following are instructions for ingress enabled, which is the recommended configuration. For instructions on non-ingress setups, see [Using Encryption and Connection](../../scst-store/using-encryption-and-connection.hbs.md) for more details.
+   
+    {{> 'partials/insight-ingress-configuration' }}
+   
+1. Configure access tokens
+
+   {{> 'partials/insight-set-access-token' }}
```

Comment:
By embedding the partials here, I think this page will be much longer.

Looking at the current overview page, it is more of an "table of contents" type page. It's summary of all the pages up ahead for the user, not the instructions themselves. It's better to add a link to `cli-configuration.hbs.md`.